### ISSUE TITLE
Evaluate terminal find screen reader support

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#150379]
It's been suggested in https://github.com/microsoft/vscode/issues/149882#issuecomment-1134946927 that find doesn't work well when using a screen reader.

**Comment by meganrogge** [comment]
doesn't announce anything ATM when pressing enter in the search field - perhaps now that we know the match index and total count, that would be useful?

**Comment by meganrogge** [comment]
the editor just makes a noise when enter is pressed in the find widget
